### DISCUSSION TITLE
Expo Build

### DISCUSSION THREAD

**Discussion by mariomurrent-softwaresolutions** [goatandsheep/react-native-dotenv#239] [Q&A]
Hi, 
is anyone using this package in combination with `expo build:ios or expo build:android`

Starting works fine, when the NODE_ENV is defined, but expo build does not take the NODE_ENV into account.

Any ideas?

**Comment by goatandsheep** [comment]
Take a look at this expo example :) https://github.com/goatandsheep/chatkitty-example-react-native/

**Comment by mariomurrent-softwaresolutions** [comment]
Well and how is the environment applied? Just when using expo build then production is used otherwise development?

**Reply by goatandsheep** [reply]
try release channels https://docs.expo.dev/distribution/release-channels/ and dev vs prod mode https://docs.expo.dev/workflow/development-mode/
